tox.ini

```diff
@@ -117,10 +123,12 @@ commands_pre=
     mkdir -p test-results .coverage
     core: ./wini package wandb-core --coverage --install
 commands=
-    pytest {env:CI_PYTEST_SPLIT_ARGS:} -n=8 --durations=20 --reruns 2 --reruns-delay 1 --junitxml=test-results/junit.xml --cov --cov-report=xml --no-cov-on-fail --timeout 300 {posargs}
+    importer-system-wandb: pytest {env:CI_PYTEST_SPLIT_ARGS:} -n=8 --durations=20 --reruns 2 --reruns-delay 1 --junitxml=test-results/junit.xml --cov --cov-report=xml --no-cov-on-fail --timeout 300 --wandb-second-server="true" {posargs}
+    !system: pytest {env:CI_PYTEST_SPLIT_ARGS:} -n=8 --durations=20 --reruns 2 --reruns-delay 1 --junitxml=test-results/junit.xml --cov --cov-report=xml --no-cov-on-fail --timeout 300 {posargs}
```

Comment:
i think you can just leave the default command and use the `tox -e ... -- --wandb-second-server="true"` it will be easier to define the config as well

Reply:
Oh I see because the flag only gets used in the lower conftest?  I didn't want to spin up a second server for every test

Reply:
yeah, but if you don't set the flag, it won't spin the other server. so basically if you want to run it locally you add the flag manually and in CI it uses a different code path anyways